test/e2e/suite/conformance/certificatesigningrequests/tests.go

```diff
@@ -166,7 +167,7 @@ func (s *Suite) Define() {
 				requiredFeatures: []featureset.Feature{featureset.OnlySAN, featureset.EmailSANsFeature},
 			},
 
-			"should issue a certificate that defines a CommonName and URI SAN": {
+			"should issue a certificate that defines a Common Name and URI SAN": {
```

Comment:
**nit:** You are correct, the commonly accepted typography seems to be either:
- "commonName" (the ASN.1 field name in the [RFC 5280](https://datatracker.ietf.org/doc/html/rfc5280)),
- "common name" (lower-cased) is the human-readable attribute name for the `commonName` field in the [RFC 5280](https://datatracker.ietf.org/doc/html/rfc5280))

```suggestion
			"should issue a certificate that defines a common name and URI SAN": {
```

Reply:
@maelvls I did not apply your requested change because all other tests also seem to use "Common Name".
And you also have "DNS Name" etc. that is used in these test files.
So currently I changed it to "Common Name" because all other tests also use this name.
Might be useful to check these test names in the future, but maybe this is out of scope for this PR?

Reply:
Totally agree, this is out of scope for this PR, my apologies 😅